internal/cloudapi/v2/v2.go

```diff
@@ -822,6 +824,17 @@ func (h *apiHandlers) GetComposeStatus(ctx echo.Context, id string) error {
 	}
 }
 
+func composeStatusErrorFromJobError(jobError *clienterrors.Error) *ComposeStatusError {
+	if jobError == nil {
+		return nil
+	}
+	return &ComposeStatusError{
```

Comment:
It would probably be better to re-use the jobError object here

Reply:
Eh, most of these apis is just transforming one go object into the one defined by the api generator.